doc/source/groupby.rst

```diff
@@ -62,16 +62,30 @@ See the :ref:`cookbook<cookbook.grouping>` for some advanced strategies.
 Splitting an object into groups
 -------------------------------
 
-pandas objects can be split on any of their axes. The abstract definition of
+Pandas objects can be split on any of their axes. The abstract definition of
 grouping is to provide a mapping of labels to group names. To create a GroupBy
-object (more on what the GroupBy object is later), you may do the following:
+object, see below (more on what the GroupBy object is later). A
+groupby can be applied in the following ways to a pandas object:
+
+* grouped = obj.groupby(key)
+* grouped = obj.groupby(key, axis='columns')
+* grouped = obj.groupby([key1, key2])
 
 .. code-block:: python
```

Comment:
this could be an ipython:: python block

Reply:
This has been reverted, see b80ff177293437d0a306127e407ba8c671dd6b4f